metricbeat/tests/system/test_apache.py

```diff
@@ -55,7 +55,7 @@ def test_output(self):
             time.sleep(0.5)
 
         proc = self.start_beat()
-        self.wait_until(lambda: self.output_lines() > 0)
+        self.wait_until(lambda: self.output_lines() > 0, max_timeout=20)
```

Comment:
apache has a timeout of more then 10s?

Reply:
I can't remember why I changed this, frakly. I know there was a good reason :confused: Default is 10 seconds and I raised to 20 because I probably received some timeout in some moment. If you want I can leave it in 10 again until I remember :sweat_smile:

Reply:
I prefer to go back to 10s as long as we don't know why.